### ISSUE TITLE
How to Make Your Medium Stories Exciting

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#53]
# TL;DR
Make your reader continue reading, and become true fan

# Key Takeaways

- **Open with a Bang**. That should be the very purpose of the 1st sentence. The sole purpose of the first sentence is to get you to read the second sentence. If your first sentence asks a question, incites curiosity, or even disgusts the reader, there’s a good chance he or she will move on to the next line. Your first sentence is your article’s most important. If it fails to engage the reader, then you’ve lost everything that matters.

- **Address Your Audience’s Needs**. Once your article gives the answers to the readers, they will inadvertently agree with whatever you write, however bizarre it can be. As the saying goes, never write for anyone, always write for someone.

- **Research**. Please remember that mishandling your research is also your potential downfall. A writer stewing in his own juices will be soon be forgotten by the even unforgiving reader.

- **Make it Specific**. Remember as writers, we need to be humble. It is always the writer’s fault for failing to express themselves clearly enough. It is never the reader’s fault for failing to understand anything. And the truth is that almost any writing in any subject can be expressed in a way that reasonably intelligent people can understand. If that is not happening, it just means that the writer has not taken enough pain to present it succinctly.

- **End with a Bang**. This is the final icing on the cake. Your last line is the first line that everybody remembers long after your story has been read.

# Link
How to Make Your Medium Stories Exciting. by @rsrajan1 in @thestartup_ https://link.medium.com/G2SW29rII5
